examples/clock.js

```diff
@@ -50,28 +50,29 @@ function numbers(x, y, radius) {
 
 // draw one of the clock hands
 function hand (theta, w, length, col) {
-   var stepSize = 5;
-   var widthDelta = w / (length / stepSize);
+   const stepSize = 5;
+   const widthDelta = w / (length / stepSize);
    goto(0, 0);
    angle(theta);
    colour(col.r, col.g, col.b, col.a);
-   for (var step = 0; step < length; step += stepSize) {
+   for (let step = 0; step < length; step += stepSize) {
       width(w);
       forward(stepSize);
       w -= widthDelta;
    }
 }
 
+// draw all of the hands
 function hands(hours, minutes, seconds) {
     // draw seconds hand
     hand(seconds * 6, 6, 100, {r: 255, g: 0, b: 0, a: 0.5 });
     // draw minutes hand
-    var minutesInSeconds = minutes * 60;
-    var minutesAndSeconds = minutesInSeconds + seconds;
+    const minutesInSeconds = minutes * 60;
+    const minutesAndSeconds = minutesInSeconds + seconds;
     hand(minutesAndSeconds * 0.1, 10, 100, {r: 0, g: 255, b: 0, a: 0.5 });
     // draw hours hand
-    var hoursInSeconds = ((hours % 12) * 3600);
-    var hoursAndMinutesAndSeconds = hoursInSeconds + minutesAndSeconds;
+    const hoursInSeconds = ((hours % 12) * 3600);
```

Comment:
B/c the operator `%`has the same precedence as both `*` & `/` those parens `()` aren't needed:
`const hoursInSeconds = hours % 12 * 3600;`